pages/jekyll.yml

```diff
@@ -35,8 +35,8 @@ jobs:
           cache-version: 0 # Increment this number if you need to re-download cached gems
       - name: Setup Pages
         id: pages
-        uses: actions/configure-pages@v1
-      - run: bundle exec jekyll build --baseurl ${{ steps.pages.outputs.base_path }} # defaults output to '/_site'
+        uses: actions/configure-pages@v2
+      - run: bundle exec jekyll build --baseurl "${{ steps.pages.outputs.base_path }}" # defaults output to '/_site'
```

Comment:
The quotes are important here in case `base_path` returns an empty string.

Alternatively, we could maybe use `toJson(...)`. 🤔 🤷🏻‍♂️